src/client/components/CreateSeason.jsx

```diff
@@ -10,38 +10,36 @@ import { TeamForm } from './seasonform/TeamForm';
 import { ParticipantForm } from './seasonform/ParticipantForm';
 import { MappingForm } from './seasonform/MappingForm';
 
-import './styling/create-season.css'
+import './styling/create-season.css';
 
 export const CreateSeason = () => {
     const { state } = useContext(StoreContext);
-    const [token, setToken] = useState()
-    const navigate = useNavigate();
 
-    console.log(state)
+    const navigate = useNavigate();
 
-    const competitionId = state.competitions[0].id;
+    const competitionId = state.selectedCompetition[0].id;
     const formTotalSteps = 5;
 
+    const token = localStorage.getItem(LOCAL_STORAGE.TOKEN);
+
     const [form, setForm] = useState({
         competitionId: competitionId,
-        title: "",
+        title: '',
         participants: [],
         teams: [],
-        mapping: [],
-    })
-    const [formStep, setFormStep] = useState(1)
+        positionMappings: [],
+    });
+    const [formStep, setFormStep] = useState(1);
 
     useEffect(() => {
-        const completedForm = formStep > formTotalSteps
-        if(completedForm) postForm()
-    }, [formStep])
+        const completedForm = formStep > formTotalSteps;
```

Comment:
Be careful with variable naming. Bools should be named something that indicates it's a bool, e.g. `isFormCompleted` or `hasCompletedForm`